### PR TITLE
Add docs for usage endpoint and command

### PR DESCRIPTION
**Pull request by kyhavlov**
Adds API and CLI docs for the new `operator usage instances` command (added [here](https://github.com/hashicorp/consul/pull/16205)).

### CODE REVIEW

File: website/content/commands/operator/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Subcommands:
+    instances    Display service instance usage information
+```
+
+## instances
+
+Corresponding HTTP API Endpoint: [\[GET\] /v1/operator/usage](/consul/api-docs/operator/usage#operator-usage)
+
+Retrieves usage information about the number of services registered in a given
```
Comment (im2nguyen) [inline website/content/commands/operator/usage.mdx:29]:
```suggestion
This command retrieves usage information about the number of services registered in a given
```

File: website/content/commands/operator/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+| ACL Required    |
+| --------------- |
+| `operator:read` |
+
+Usage: `consul operator usage instances`
+
+The output looks like this:
+
+```text
+Billable Service Instances Total: 3
```
Comment (im2nguyen) [inline website/content/commands/operator/usage.mdx:45]:
````suggestion
```shell-session
$ consul operator usage instances
Billable Service Instances Total: 3
````

File: website/content/commands/operator/usage.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+connect-proxy            0
+ingress-gateway          0
+mesh-gateway             1
+terminating-gateway      0
+```
+
+With the `-all-datacenters` flag:
+
+```text
+Billable Service Instances Total: 4
```
Comment (im2nguyen) [inline website/content/commands/operator/usage.mdx:64]:
````suggestion
```shell-session
$ consul operator usage instances -all-datacenters
Billable Service Instances Total: 4
````

**Review by im2nguyen — APPROVED** [review]

**Review by ishustava — APPROVED** [review]
LGTM. +1 to Tu's suggestions.

**Review by david-yu — APPROVED** [review]
Looks good to me with Tu's suggestions.

**Pull request merged**